### PR TITLE
676 remove error paths

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

File: web-client/src/presenter/sequences/autoSaveCaseSequence.js [hunk 1]
```diff
@@ -9,13 +8,7 @@ export default [
...
   {
-    success: [
-      updateCase,
-      {
-        error: [setAlertError],
-        success: [],
-      },
-    ],
+    success: [updateCase],
     error: [],
```
Comment (kkoskelin) [inline web-client/src/presenter/sequences/autoSaveCaseSequence.js:12]: Do we need 'error'?

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/presenter/sequences/autoSaveCaseSequence.js (hunk 1 above)
Comment (ericsorenson) [reply web-client/src/presenter/sequences/autoSaveCaseSequence.js:12]: actually yes, turns out

**Review by sadlerw — APPROVED** [review]

**Comment by sadlerw** [comment]
Let's just keep in mind that the errors thrown by the interactors (business errors) do need to be caught in the actions and handled there, or some mechanism needs to be in place to allow the custom errors from the interactors to be handled per the business requirements

**Pull request merged**